### ISSUE TITLE
`hlint` should run on the latest non-GHCHEAD version

### ISSUE THREAD

**Issue by RyanGlScott** [haskell-CI/haskell-ci#270]
In a `.travis.yml` file generated with `haskell-ci-0.3.20190425`, I see the following:

```
  # HLint..
- if [ $HCNUMVER -eq 80801 ] ; then (cd heaps-* && hlint --cpp-ansi --cpp-define=HLINT src) ; fi
```

Given that GHC 8.8.1 is still pre-release, I'm quite surprised to see that _that_ was the version picked for `--hlint`. Would it make more sense to pick the most recent, non-`GHCHEAD` version of GHC instead so that 8.8.1 isn't considered?

**Comment by phadej** [comment]
Good catch

**Comment by RyanGlScott** [comment]
Fixed in #271.
